packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -568,27 +565,32 @@ export default class PciInstancesAddController {
     this.addons = [];
     if (this.isPrivateMode() && this.isAttachFloatingIPAvailable) {
       this.isLoadBillingStep = true;
-      this.getSubnetGateways(this.selectedPrivateNetwork.subnet[0].id)
-        .then((data) => {
-          this.subnetGateways = data;
+      if (this.selectedPrivateNetwork?.isCustom) {
```

Comment:
Hi @JeremyDec , I have added the logic on L566 as suggested by you. But in the below line where I am checking isCustom is required because its an optional param which could be there if PrivateNetwork is created using modal.